_docs/index.md

```diff
@@ -10,6 +10,8 @@ type: markdown
 ---
 {% include home.html %}
 
+Follow this link for the archive of [most recent stable release (0.1) documentation](https://istio.io/v-0.1/docs/).
```

Comment:
👍 on making it more proeminent (we may need to put it in all page nave if we could)